services/static-webserver/client/source/class/osparc/service/StatusUI.js

```diff
@@ -110,6 +110,19 @@ qx.Class.define("osparc.service.StatusUI", {
       }
     },
 
+    getStatusHalo: function(element, progressColor, progress) {
+      console.log("element", element.getHeight())
+      // element.set({
```

Comment:
are you guys ok leaving commented code?

Reply:
Usually not, and ``console.log``s neither, but I'll take over, since he will be away for a few days 👍